tests/neg/equality.scala

```diff
@@ -0,0 +1,61 @@
+import dotty.{Eq,EqClass}
+
+object equality {
+
+  case class Str(str: String) extends EqClass[_]
```

Comment:
Shouldn't this be `EqClass[Str]`? I'm not even sure what `extends EqClass[_]` means, that syntax is not allowed in scala 2.